p2p_beef_tx.go

```diff
@@ -123,8 +123,8 @@ func decodeBUMPPathsFromStream(hexBytes []byte) ([][]BUMPLeaf, []byte, error) {
 		return nil, nil, errors.New("cannot decode BUMP paths from stream - no bytes provided")
 	}
 
-	treeHeight, bytesUsed := bt.NewVarIntFromBytes(hexBytes)
-	hexBytes = hexBytes[bytesUsed:]
+	treeHeight := hexBytes[0]
```

Comment:
Why did you changed treeHeight from VarInt to bytes?

AFAIU max number that can be stored in 1 byte is 255.
Tree of height 255 can contain 65025 transactions.
And we're aiming on 1 mln of transactions and more.

Reply:
I think because in documentation treeHeight size equals 1 byte
https://bsv.brc.dev/transactions/0074#global

Reply:
![image](https://github.com/bitcoin-sv/go-paymail/assets/135072995/dd3a6984-3ae8-4a6d-ab4b-e56a7e36cb3d)

Reply:
I believe we do not want to send 1 mln transactions in BUMP